test_cases/test_dojo.py

```diff
@@ -239,6 +243,39 @@ def test_load_people_invalid_entry(self):
 		result = self.the_dojo.load_people("LoadFile")
 		self.assertEqual(result, "Invalid entry!")	
 
+	def test_save_state_database_file_exists(self):
+		self.the_dojo.save_state("sampleDB")
+		self.assertTrue(os.path.isfile("sampleDB.db"))
+		os.remove("sampleDB.db")
 
+	def test_save_and_load_room_data_to_DB(self):
+		self.the_dojo2 = Dojo()
+		self.the_dojo.create_room("office", "Red")
+		self.the_dojo2.create_room("livingspace", "Ruby")
+		self.the_dojo.save_state("sampleDB")
+		
+		self.the_dojo2.load_state("sampleDB")
+		self.assertEqual(len(self.the_dojo2.all_offices), 1)
+		self.assertEqual(len(self.the_dojo2.all_livingspace), 1)
+
+	def test_save_and_load_persons_data_to_DB(self):
+		self.the_dojo2 = Dojo()
+		self.the_dojo.add_person("Pete","pete@pete","staff")
+		self.the_dojo.add_person("Tye", "ty@ty", "fellow", "y")
+		self.the_dojo.save_state("sampleDB")
+
+		self.the_dojo2.load_state("sampleDB")
+		self.assertEqual(len(self.the_dojo2.all_fellows), 1)
```

Comment:
The same here length of all people added!
Just a suggestion you could try having  both rooms created and people added tests in one test just to make your code more DRY